maven/lib/dependabot/maven/file_updater.rb

```diff
@@ -40,18 +43,18 @@ def updated_dependency_files
       private
 
       def check_required_files
-        raise "No pom.xml!" unless get_original_file("pom.xml")
+        raise "No pom.xml!" unless get_original_file("pom.xml") || get_original_file("extensions.xml")
```

Comment:
This should probably be done differently.

Reply:
Can you have a `extensions.xml` file without a `pom.xml`? Wondering if this should be left as it was, and only requiring `pom.xml` leaving extensions optional

Reply:
You could have an extension that replaces `pom.xml` with something else. [polyglot-maven](https://github.com/takari/polyglot-maven) for example lets you write the build with languages other than xml. So could have a `pom.groovy` instead.

Reply:
Gotcha, although dependabot only supports `pom.xml` atm. I think it would be better to fail in the case where there's a `pom.groovy` and a `estensions.xml` as dependabot would ignore the groovy file and update the extensions file, which is probably not what you expect.

I think we should revert this change unless there's a way to have a `extensions.xml` file without a `pom.*` file.

Reply:
@britter thoughts on reverting this to always require a `pom.xml`?

Reply:
@feelepxyz sorry, I missed this comment. I'm fine with reverting. I think would be possible to write an extension that makes a Maven build work without any other build file, but I don't think it's a relevant use case for this feature.